mixins/_typography.pcss

```diff
@@ -206,6 +206,11 @@
  * Heading 6
  * ----------------------------------------------------------------------------- */
 
+@define-mixin mobile-heading-6 {
+	font-size: var(--font-size-3);
+	line-height: var(--line-height-2);
+}
+
 @define-mixin heading-6 {
```

Comment:
since you're using `desktop-heading-6` in common, this should be updated.

Reply:
Thanks! Nice catch